src/cosmos-copilot.WebApp/Program.cs

```diff
@@ -1,12 +1,55 @@
+using Azure.Identity;
 using Cosmos.Copilot.Options;
 using Cosmos.Copilot.Services;
+using Microsoft.Azure.Cosmos;
 using Microsoft.Extensions.Options;
 
 var builder = WebApplication.CreateBuilder(args);
 
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+
 builder.RegisterConfiguration();
 builder.Services.AddRazorPages();
 builder.Services.AddServerSideBlazor();
+
+// Configure Azure Cosmos DB Aspire integration
+builder.AddAzureCosmosClient(
+    "cosmos",
+    settings =>
+    {
+        settings.Credential = new DefaultAzureCredential();
+        settings.DisableTracing = false;
+    },
```

Comment:
add this line to settings 

settings.AccountEndpoint = new Uri(cosmosEndpoint!);